pontoon/sync/vcs/models.py

```diff
@@ -623,6 +623,10 @@ def __init__(self, vcs_project, path, locales=None):
                 string=translation.source_string,
                 string_plural=translation.source_string_plural,
                 comments=translation.comments,
+                group_comments=translation.group_comments if hasattr(
+                    translation, 'group_comments') else None,
+                resource_comments=translation.resource_comments if hasattr(
+                    translation, 'resource_comments') else None,
```

Comment:
Matjaz, correct me if I'm wrong but there's no need for the condition here, as `group_comments` and `resource_comments` are defined in `VCSTranslation`. Thus, every instance of `translation` should have them, even is set to `None`.

Reply:
I'm afraid it's needed afterall. :/

While for example `SilmeEntity` inherits from `VCSTranslation`, it doesn't initialize its attributes (with `super()`). To do that, we'd need to refactor pretty much every file format quite a bit, because we use the `@property` decorator, which doesn't play nicely with `super()`. Or maybe you have an idea how to do that more easily?

Reply:
Matjaz,  I did have these as one line originally but the linter failed as it was over the 100 character limit. Is there a preferred way for handling this?

Reply:
Good point, in this case let's do this:
```python
group_comments=(
    translation.resource_comments
    if hasattr(translation, 'resource_comments') else None
),
```

Reply:
Nope, I don't know the code enough to propose anything else. Forget that I was there! :smile: